pouta_blueprints/views/instances.py

```diff
@@ -74,15 +75,8 @@ class InstanceList(restful.Resource):
     def get(self):
```

Comment:
This function calls a lot of helper functions so it might be helpful for the next reader to have a simplified human-readable explanation of what instances a user or admin is allowed to see.